.github/scripts/build_assets/arg_getters.py

```diff
@@ -36,7 +36,7 @@ def get_selenium_runner_args(peek_mode=False):
     if peek_mode:
         parser.add_argument("pr_title",
                             help="The title of the PR that we are peeking at")
-    else:
+    elif has_token != False:
```

Comment:
> Also, if one excludes the other, why do we need two variables?

Could you explain this, please? Not 100% sure what you mean.

> what happens if `has_token` and `peek_mode` is false?
> Seems like the method won't return anything then, will that cause issues?

Good question. I'll add the token on an `else` statement, as the token is required, while the PR title is not, I think.

Reply:
> > Also, if one excludes the other, why do we need two variables?
>
> Could you explain this, please? Not 100% sure what you mean.

What is the `peek_mode` variable for? Seems to me like it only does the opposite of `has_token`. If it's true, then the PR title argument will be added, if it's not true, then the token argument will be added. If `has_token` is true, then the opposite will happen.

Shouldn't they either be unrelated (e.g. two if statements instead of if-else), or just be one variable?

Reply:
I edited the message by the way.

Reply:
Wouldn't it be better to change it from elseif to just an if then?

Reply:
Can't it just be as simple as this?
```suggestion
    if has_token:
```